Lekce_02_Ukol02/Program.cs

```diff
@@ -4,20 +4,30 @@ internal class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("Zadej počet hvězdiček: ");
+            string vstup = Console.ReadLine();
+
             try
             {
-                Console.WriteLine("Zadej počet hvězdiček: ");
-                string vstup = Console.ReadLine();
                 int pocetHvezdicek = int.Parse(vstup);
 
+                if (pocetHvezdicek < 0)
+                {
+                    throw new ArgumentOutOfRangeException("Počet hvězdiček nesmí být záporný");
+                }
+
                 for (int i = 0; i < pocetHvezdicek; i++)
```

Comment:
nit: Tenhle cyklus by se taky mohl posunout mimo ten `try` blok :)

Reply:
opraveno

Reply:
Pardon, jak je napsany tenhle kod, tak ten for cyklus by mel byt v tom try catchi po tech if kondicich 🙏 Moje chyba 😅